frontend/packages/console-shared/src/components/status/PopoverStatus.tsx

```diff
@@ -8,6 +9,8 @@ const PopoverStatus: React.FC<PopoverStatusProps> = ({
   icon,
   activeIcon,
   children,
+  isVisible = null,
+  shouldClose = null,
```

Comment:
Do we need to default to null? I'd think `undefined` would be OK

Reply:
There is a conditional check in the underlying component which will ignore the isVisible prop as long as it `=== null`. I have a feeling that letting it default to undefined might cause some undesired behavior.

Reply:
That almost sounds like a bug in the PF component. It treats `null` and `undefined` differently?